chain/jsonrpc/src/lib.rs

```diff
@@ -30,7 +30,7 @@ use near_jsonrpc_client::ChunkId;
 use near_metrics::{Encoder, TextEncoder};
 #[cfg(feature = "adversarial")]
 use near_network::types::NetworkAdversarialMessage::{
-    AdvDisableHeaderSync, AdvGetSavedBlocks, AdvProduceBlocks, AdvSetSyncInfo,
+    AdvDisableDoomslug, AdvDisableHeaderSync, AdvGetSavedBlocks, AdvProduceBlocks, AdvSetSyncInfo,
```

Comment:
I want to say that importing the every single enum variant into the "global" namespace looks a bit ugly to me. Should we only import enums and always namespace the enum variants with the enum variant name instead of having "Adv" prefix?

Reply:
perfectly makes sense